.gitignore

```diff
@@ -1,7 +1,9 @@
 # stellar/soroban/Rust output
 target
 .soroban
-.stellar
+
```

Comment:
Stellar CLI is no longer using this folder, and we set XDG_CONFIG in .env to ".config" so that ".config/stellar" gets used for identities and aliases.